doc/releases/release-notes-2.2.rst

```diff
@@ -429,7 +430,21 @@ Bluetooth
 
 * Host:
 
-  * <TBD>
+  * GAP: support for Extended Advertising
+  * GAP: Add dynamic LE scan listener API
+  * GAP: Pre-allocate connection objects for connectable advertising
+  * GAP: Fixes for multi-identity support
+  * GAP: RPA timeout handling fixes
+  * L2CAP: move channel processing to the system workqueue
+  * L2CAP: multiple fixes for credit-based flowcontrol
+  * SMP: Fix Security Manager timeout handling
```

Comment:
I think we could add the following:
SMP: Add pairing_accept callback.
GATT: Add return value to cfg_write callback.
GAP: Add remote version information.

Reply:
Added now